pkg/controller/atlasdatabaseuser/atlasdatabaseuser_controller.go

```diff
@@ -202,6 +202,10 @@ func (r *AtlasDatabaseUserReconciler) ready(ctx *workflow.Context, atlasDatabase
 		EnsureStatusOption(status.AtlasDatabaseUserNameOption(atlasDatabaseUser.Spec.Username)).
 		EnsureStatusOption(status.AtlasDatabaseUserPasswordVersion(passwordVersion))
 
+	if atlasDatabaseUser.Spec.ExternalProjectRef != nil {
+		return workflow.Requeue(workflow.StandaloneResourceRequeuePeriod).ReconcileResult()
+	}
+
 	return workflow.OK().ReconcileResult()
```

Comment:
This is the only place we exit OK right?

Reply:
Yes